setup.py

```diff
@@ -51,18 +51,13 @@ def required(requirements_file):
         return [pkg for pkg in requirements
                 if pkg.strip() and not pkg.startswith("#")]
 
+with open("README.md", "r") as f:
```

Comment:
This should be updated to an absolute path for automations, see https://github.com/OpenVoiceOS/ovos-bus-client/pull/95